OpenSim/Simulation/Model/PhysicalOffsetFrame.h

```diff
@@ -53,28 +53,15 @@ class OSIMSIMULATION_API PhysicalOffsetFrame : public OffsetFrame<PhysicalFrame>
     //--------------------------------------------------------------------------
     // CONSTRUCTION
     //--------------------------------------------------------------------------
-    /** By default, the frame is not connected to any parent frame,
-     * and its transform is an identity transform.
-     */
-    PhysicalOffsetFrame();
+    /** Constructors are defined by the OffsetFrame base class */
+    using OffsetFrame<PhysicalFrame>::OffsetFrame;
 
-    virtual ~PhysicalOffsetFrame() {}
-
-    /**
-    A convenience constructor that initializes the parent connection and
-    offset property of this PhysicalOffsetFrame.
-
-    @param[in] parent   The parent PhysicalOffsetFrame.
-    @param[in] offset   The offset transform between this frame and its parent
-    */
-    PhysicalOffsetFrame(const PhysicalFrame& parent,
-                        const SimTK::Transform& offset);
+    ~PhysicalOffsetFrame() final {}
 
 protected:
     /** Extend Component interface for adding the PhysicalOffsetFrame to the 
         underlying multibody system */
-    void extendAddToSystem(SimTK::MultibodySystem& system) const override;
-
+    void extendAddToSystem(SimTK::MultibodySystem& system) const override final;
```

Comment:
`PhysicalOffsetFrame` is a mixin (i.e.. `OffsetFrame<PhysicalFrame>`) so I would want `OffsetFrame<>` to be extended (including more convenient constructors). It is for that particular reason (where i almost added a new convenience constructor to PhysicalOffsetFrame instead of OffsetFrame<>) that I wish to block extensions here. In hindsight that may be too harsh and even impractical, but that was my reasoning. I can add that reasoning to the class description and/or eliminate the restriction.

What is your preference here? @sherm1  @chrisdembia @aymanhab